build/Build.cs

```diff
@@ -199,14 +207,49 @@ class Build : NukeBuild
 
     void SignBinaries(AbsolutePath path)
     {
-        Info($"Signing binaries in {path}");
-        var files = path.GlobDirectories("**").SelectMany(x => x.GlobFiles("Octopus.*.dll"));
+        Logger.Info($"Signing binaries in {path}");
+        var files = path.GlobDirectories("**").SelectMany(x => x.GlobFiles("Octopus.*.dll")).ToArray();
+
+        if (string.IsNullOrEmpty(AzureKeyVaultUrl)
+            && string.IsNullOrEmpty(AzureKeyVaultAppId)
```

Comment:
This feels like these should be `||` rather than `&&`. If any of those things are blank then `SignWithAzureSignTool` isn't going to work, is it?

Reply:
This is a direct clone of what's in server which has been working happily.
If any of these are blank, azuresigntool will error with a nice error message (theoretically at least), so I'm not massively concerned here.

Reply:
Fair enough 👍